nav2_lifecycle/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
```

Comment:
We're using format 3 everywhere now.

Reply:
Fixed in nav2_controller and nav2_lifecycle